### PR TITLE
example of loading an obj file

### PR DESCRIPTION
**Pull request by sirkitree**
This file doesn't exist so we may want an example .obj file as well to go along with this. We should also illustrate how to load the .mtl file along with this but I'm not sure how to do that just yet...

### CODE REVIEW

File: src/docs/primitives/a-model.md [hunk 1]
```diff
@@ -30,3 +30,8 @@ Using scale to increase to size of the loaded tree model:
 <a-model scale="10 10 10" src="https://aframe.io/aframe/examples/_models/tree1/tree1.dae"></a-model>
 ```
 
+Loading an obj formatted file:
```
Comment (ngokevin) [inline src/docs/primitives/a-model.md]: maybe like: "`obj`-formatted"

File: src/docs/primitives/a-model.md [hunk 2]
```diff
@@ -30,3 +30,8 @@ Using scale to increase to size of the loaded tree model:
 <a-model scale="10 10 10" src="https://aframe.io/aframe/examples/_models/tree1/tree1.dae"></a-model>
 ```
 
+Loading an obj-formatted file:
+
+```html
+<a-model src="https://aframe.io/aframe/examples/_models/tree.obj" format="obj"></a-model>
```
Comment (cvan) [inline src/docs/primitives/a-model.md]: we have https://github.com/aframevr/aframe-core/blob/master/examples/model/models/teapot.obj

Comment (cvan) [inline src/docs/primitives/a-model.md]: @jcarpenter do you want to add some `.obj`s to `examples/` in this repo?

Comment (jcarpenter) [inline src/docs/primitives/a-model.md]: Yep, good call. Not a high priority, but something we should do.

Comment (cvan) [inline src/docs/primitives/a-model.md]: I uploaded it `teapot.obj` here, btw: http://www.ucarecdn.com/c1ba381b-842a-4d9b-af8e-a14b353b22e8/teapot.obj so we could use that

**Comment by ngokevin** [comment]
Thanks for this. We haven't versioned our docs yet so we couldn't release this yet. I've documented this for the next version, and we'll have a better way to contrib to docs.

**Pull request closed without merging**